empack_config.yaml

```diff
@@ -0,0 +1,42 @@
+packages:
+  python-dateutil:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - pattern: '*dateutil-zoneinfo.tar.gz'
+    exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+  matplotlib:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - pattern: '*matplotlibrc'
```

Comment:
Actually this one is already included implicitely in the line below (under `mpl-data`), we can remove it

```suggestion
```